web/client/plugins/Identify.jsx

```diff
@@ -133,14 +133,18 @@ const identifyDefaultProps = defaultProps({
 });
 
 /**
- * Identify plugin. This plugin allows to perform get feature info.
- * It can be configured to have a mobile or a desktop flavor.
- * It's enabled by default. The bubbling of an on_click_map action to GFI is stopped
- * if Annotations or FeatureGrid plugins are editing, draw or measurement supports are
- * active, the query panel is active or the identify plugin is disabled.
- * To restore old behavior, in mapInfo state, set disabledAlwaysOn to true and
- * manage the plugin using toggleControl action with 'info' as control name.
- * It's possible also possible disable the plugin by changeMapInfoState or toggleMapInfoState actions
+ * This plugin allows get informations about clicked point. It can be configured to have a mobile or a desktop flavor.
+ *
+ * You can configure some of the features of this plugin by setting up the initial mapInfo state t you need to update the "initialState.defaultState", or by the plugin configuration
+ * ```
```

Comment:
between `state` and `you` there is a `t`, probably is `then`

Reply:
fixed